__main__.py

```diff
@@ -5,7 +5,7 @@
 from github import GithubClient
 from jira import JiraClient
 from testrail import TestRailClient
-from utils.constants import PROJECTS_MOBILE, PROJECTS_ECOSYSTEM, PLATFORM, REPORT_TYPES # noqa
+from utils.constants import PROJECTS_MOBILE, PROJECTS_ECOSYSTEM, PROJECTS_DESKTOP, PLATFORM, REPORT_TYPES # noqa
```

Comment:
`PLATFORM` should be `PLATFORMS` sign it will be assigned more than 1 value by definition.  However, when you assign a local variable with the platform value that variable would be singular:  `platform`

Reply:
changed here and in constants